scalableminds/webknossos
# Fix rendering artifacts in tasks that only allow flight mode

Objects that should only be visible in orthogonal mode were by default always visible after their initialization. They are hidden when stopping orthogonal mode, but that didn't happen if orthogonal mode was never active.

I fixed this by hiding all those objects after their initialization, they'll be made visible when orthogonal mode is started anyways.

If you have a better name, for start/stopPlaneMode, please let me know :)

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Create a task type that only allow flight mode and create a task with that task type.
- Request said task and open it - There should be no rendering artifacts visible (see screenshot in issue).
- Open an explorative tracing (which starts in orthogonal mode by default). Everything should be visible as before.

### Issues:
- fixes #3776 

------
